src/lsm/schema.zig

```diff
@@ -343,13 +344,12 @@ pub const TableData = struct {
     }
 };
 
-pub const FreeSetNode = struct {
-    pub const Word = u64;
-
+/// A TrailerNode is either a `BlockType.free_set` or `BlockType.client_sessions`.
+pub const TrailerNode = struct {
     pub const Metadata = extern struct {
-        previous_free_set_block_checksum: u128,
-        previous_free_set_block_checksum_padding: u128 = 0,
-        previous_free_set_block_address: u64,
+        previous_trailer_block_checksum: u128,
+        previous_trailer_block_checksum_padding: u128 = 0,
```

Comment:
Wait, do we need this padding? I think we switched all pure checksumms to be unpadded?

Reply:
I don't think this actually is a pure checksum -- it is referring to a block, which would be encrypted, so the "checksum" would be the AEAD tag.